bundles/org.openhab.core.karaf/src/main/java/org/openhab/core/karaf/internal/FeatureInstaller.java

```diff
@@ -302,11 +332,13 @@ private boolean setOnlineStatus(boolean status) {
                     if (!repoCfg.contains(online_repo_url)) {
                         repoCfg.add(online_repo_url);
                         changed = true;
+                        logger.debug("Added repo '{}' to feature repo list.", online_repo_url);
```

Comment:
Not new, but

`online_repo_url` => does not follow naming conventions.

Reply:
right, not new.